ynr/apps/sopn_parsing/helpers/convert_pdf.py

```diff
@@ -42,6 +42,3 @@ def convert_sopn_to_pdf(uploaded_file):
             uploaded_file.file = ContentFile(content=temp_file.file.read())
             uploaded_file.name = pdf_file_name
             return uploaded_file
```

Comment:
I removed the `ValueError` bc it was breaking several unrelated tests and I wasn't sure they should be changed in any way. Also, would we even hit the `else` since there is now validation added on the form?

Reply:
I wonder if some of the failing tests could have been related to https://github.com/DemocracyClub/yournextrepresentative/pull/1710/files/063c28e92b1d2f25a0791fcfb5773ca4f180015c#r798550011

I'll check out locally to try it out, as I mean to add that change as per Syms suggestion anyway